### PR TITLE
quickstart intro

### PR DESCRIPTION
**Pull request by Ben-Rey**

### CODE REVIEW

File: docs/build/quickstart.mdx [hunk 1]
```diff
@@ -3,14 +3,25 @@ id: quickstart
 sidebar_label: Quickstart
 ---
 
-# Build your first decentralized application
+# QuickStart
 
-TODO: Explain what we will do: build a hello world app
+---
```
Comment (qdrn) [inline docs/build/quickstart.mdx:8]: I'd remove this line, it looks weird.

File: docs/build/quickstart.mdx [hunk 2]
```diff
@@ -3,14 +3,25 @@ id: quickstart
...
+
+In this guide, we're going to build a simple "Hello world" decentralized application (dApp) on the Massa blockchain. This application will contain a smart contract written in AssemblyScript and a basic frontend using html, css, javascript or React, based on your preference.
+
+### **The Application Overview**
+
+Our application will be straightforward - a user can change a greeting message stored on the blockchain, and any changes will be permanent and visible to all users of the dApp.
+
+- **Smart Contract (SC)**
+
+The smart contract is the backbone of our application. It is a self-executing contract with the terms of the agreement directly written into lines of code. We will use AssemblyScript to write our smart contract. This contract will hold a single greeting message that can be updated and read.
```
Comment (qdrn) [inline docs/build/quickstart.mdx:20]: I think `self-executing` might be confused with autonomous SCs in our case. Might be better to find an other term.

File: docs/build/quickstart.mdx [hunk 3]
```diff
@@ -3,14 +3,25 @@ id: quickstart
 sidebar_label: Quickstart
 ---
 
-# Build your first decentralized application
+# QuickStart
```
Comment (qdrn) [inline docs/build/quickstart.mdx:6]: I think it was better before. The structure made more sense. Something like this:
```md
# Build your first decentralized application

In this guide...

## **The Application Overview**

blabla

## Smart Contract

blabla
...
```

**Review by qdrn — COMMENTED** [review]
Overall it sounds good to me. I've made small comments. Feel free to ignore as I may take time to answer questions.

**Review by gregLibert — APPROVED** [review]

**Pull request merged**